cardano-crypto-class/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Changelog for `cardano-crypto-class`
 
-## 2.2.3.1
+## 2.2.4.0
```

Comment:
This is a patch version update. There are no semantic changes to the library itself